### DISCUSSION TITLE
Micropython

### DISCUSSION THREAD

**Discussion by thestumbler** [FYSETC/FYSETC-E4#24] [Q&A]
I found this board by accident, but think it would be perfect for a non-3D-printer project of mine.
Has anyone built MicroPython for this board already?
I'm equally (or more) comfortable in C/C++, but I have already coded a bunch of prototype software and proof-of-concept demo software for my project in MicroPython. I also have some experience in configuring custom boards for MicroPython, if nobody has already done this.
-Chris

**Comment by mcgheealex** [comment]
I would also like to have micropython support, but have no idea how to do that. Any updates on this front?
